modular_mithra/code/modules/mithra/traits_neutral.dm

```diff
@@ -58,4 +76,92 @@
+		H.setMaxHealth(S.total_health)
+
+
+/datum/trait/minor_brute_weak
+	name = "Thin Skin"
+	desc = "Your skin is thinner than normal, making you slightly more susceptible to physical damage."
+	var_changes = list("brute_mod" = 1.50)
+
+/datum/trait/minor_burn_weak
+	name = "Inflammable Skin"
+	desc = "You skin is somehow inflammable, making you slightly more susceptible to burns."
+	var_changes = list("burn_mod" = 1.50)
+
+/datum/trait/conductive
+	name = "Conductive Skin"
+	desc = "Your skin has a lower electrical resistivity than normal, making you much more conductive."
+	var_changes = list("siemens_coefficient" = 1.5) //This makes you a lot weaker to tasers.
+
+
+/datum/trait/photosensitive
+	name = "Photosensitive"
+	desc = "Your eyes are especially sensitive to bright lights."
+	var_changes = list("flash_mod" = 2.0)
+
+/datum/trait/hollow
+	name = "Glass Bones"
+	desc = "Your limbs are just more fragile than usual for whatever reason, making them easier to break."
+
+/datum/trait/hollow/apply(var/datum/species/S,var/mob/living/carbon/human/H)
+	..(S,H)
+	for(var/obj/item/organ/external/O in H.organs)
+		O.min_broken_damage *= 0.5
+
+/datum/trait/oxy_weak
+	name = "Haemoglobin Improbus"
+	desc = "You have a lower blood-oxygen saturation, meaning asphyxiation is a greater risk to you."
+	var_changes = list("oxy_mod" = 1.50)
+
+/datum/trait/toxin_weak
+	name = "Paper Liver"
+	desc = "Your metabolism isn't very good at processing toxins, making poisons more effective against you."
+	var_changes = list("toxins_mod" = 1.50)
+
+/datum/trait/noodlyarms
+	name = "Muscular Atrophy"
+	desc = "You have less muscle mass than normal, giving you inferior strength."
+	var_changes = list("strength" = STR_LOW)
+
+/////////////////////
+// BoH Materials
+/////////////////////
+
+/datum/trait/toxification_junky
+	name = "Weak Immune System"
+	desc = "Your immune system is incredibly weak, and even the slightest ailment may kill you. Be sure to notify Doctors not to overdose you!"
+	var_changes = list("toxins_mod" = 5.0)
+
+/datum/trait/hemophilia
+	name = "Hemophilia"
+	desc = "Your body doesn't quite stop bleeding once it starts. You need immediate treatment for anything, even minor wounds, or it might turn out real bad for you."
+	var_changes = list("blood_volume" = 105) //The bare minimum before it becomes critical.
+
+/datum/trait/stick_human
+	name = "Incredibly Frail"
+	desc = "Your bones, skin and general state of mind is rather fragile. Try not to get smacked, or you may have to visit the ER."
+	var_changes = list("brute_mod" =2.50)
+
+	/datum/trait/commune
+	name = "Telepathy"
+	desc = "Quite simply, you've the ability to project thoughts into the minds of others. A weak psychic manifestation too minor to require action from the local authorities, unlikely to ever develop into something greater."
+/datum/trait/commune/apply(var/datum/species/S,var/mob/living/carbon/human/H)
+	..(S,H)
+	H.verbs |= /mob/living/carbon/human/proc/psychic_whisper
```

Comment:
We talked about that, there's no purpose for it's existance other than abuse, it waas abused in the past and really adds nothing positive.

Reply:
right